airbyte-webapp/src/components/EntityTable/components/StatusCell.tsx

```diff
@@ -35,7 +35,10 @@ const StatusCell: React.FC<IProps> = ({ enabled, isManual, id, onChangeStatus, i
   };
 
   if (!isManual) {
-    return <Toggle checked={enabled} onChange={OnToggleClick} disabled={!allowSync} />;
+    // Getting the loading state down here may be difficult.
+    // ConnectionsTable.tsx#28
+    // The data is invalidated, then re-loaded. There's no specific action flow for loading.
```

Comment:
@edmundito I think this should rather just be a coment on the PR not in the code. Could we remove this before merging this?